x/poe/bootstrap.go

```diff
@@ -91,14 +91,48 @@ func bootstrapPoEContracts(ctx sdk.Context, k wasmtypes.ContractOpsKeeper, tk tw
 	}
 	ocInitMsg := newOCInitMsg(gs)
 	deposit := sdk.NewCoins(gs.OversightCommitteeContractConfig.EscrowAmount)
-	ocContractAddr, _, err := k.Instantiate(ctx, ocCodeID, systemAdminAddr, systemAdminAddr, mustMarshalJson(ocInitMsg), "oversight_committee", deposit)
+
+	firstMember, err := sdk.AccAddressFromBech32(gs.OversightCommunityMembers[0])
+	if err != nil {
+		return sdkerrors.Wrap(err, "first member")
+	}
+
+	ocContractAddr, _, err := k.Instantiate(ctx, ocCodeID, firstMember, systemAdminAddr, mustMarshalJson(ocInitMsg), "oversight_committee", deposit)
 	if err != nil {
 		return sdkerrors.Wrap(err, "instantiate tg trusted circle contract")
 	}
 	poeKeeper.SetPoEContractAddress(ctx, types.PoEContractTypeOversightCommunity, ocContractAddr)
 	if err := k.PinCode(ctx, ocCodeID); err != nil {
 		return sdkerrors.Wrap(err, "pin tg trusted circle contract")
 	}
+
+	if len(gs.OversightCommunityMembers) > 1 {
+		tcAdapter := contract.NewTrustedCircleContractAdapter(ocContractAddr, tk, nil)
+		err = tcAdapter.AddVotingMembersProposal(ctx, gs.OversightCommunityMembers[1:], firstMember)
+		if err != nil {
+			return sdkerrors.Wrap(err, "add voting members proposal")
+		}
+		latest, err := tcAdapter.LatestProposal(ctx)
+		if err != nil {
+			return sdkerrors.Wrap(err, "query latest proposal")
+		}
+		err = tcAdapter.ExecuteProposal(ctx, latest.ID, firstMember)
+		if err != nil {
+			return sdkerrors.Wrap(err, "execute proposal")
+		}
+		// deposit escrow
+		for _, member := range gs.OversightCommunityMembers[1:] {
+			ocMember, err := sdk.AccAddressFromBech32(member)
+			if err != nil {
+				return sdkerrors.Wrapf(err, "%s oc member", member)
+			}
+			err = tcAdapter.DepositEscrow(ctx, gs.OversightCommitteeContractConfig.EscrowAmount, ocMember)
+			if err != nil {
+				return sdkerrors.Wrap(err, "deposit escrow")
```

Comment:
nit: please add the address to the error msg as well. If there are a lot this will help finding the issue.